recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-extended/xt-log/xt-log_git.bbappend

```diff
@@ -1,2 +1,3 @@
 SRC_URI = "git://github.com/xen-troops/${PN}.git"
-SRCREV = "${AUTOREV}"
+SRCREV = "152eaf422b65eecc5bee63521e4606be21d2cb27"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=1ebbd3e34237af26da5dc08a4e440464"
```

Comment:
As I understand, we do not need LIC_FILES_CHKSUM to be specified here.
I have checked that we have meta-xt-images fixed in meta-xt-products to version with this checksum (for GPL3).
And xt-log is fixed right here to same GPL3 version.
So, there is no need for LIC_FILES_CHKSUM.

Reply:
I agree, but build required it, it was not my idea :(
anyway i will recheck

Reply:
I have checked. This LIC_FILES_CHKSUM is defined in revision
1a19ac5 Add C++ header only log
of file https://github.com/xen-troops/meta-xt-images/commits/master/recipes-domx/meta-xt-images-domx/recipes-extended/xt-log/xt-log_git.bb
We have xt-images 'fixed' to that resision in product's manifest, so no need to redefine it here.

I have built tu2019 with this pull request with removed line "LIC_FILES_CHKSUM ...".
DomD built succesfully.

So, could you please remove line with checksum and merge this PR.

Reply:
ok